pkg/kubernetes/version_test.go

```diff
@@ -0,0 +1,86 @@
+package kubernetes_test
+
+import (
+	"github.com/billiford/go-clouddriver/pkg/kubernetes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var (
+	kc                kubernetes.Controller
+	fakeResourcesList *unstructured.UnstructuredList
+	currentVersion    string
+)
+
+var _ = Describe("Pkg/Kubernetes/Version", func() {
+	Context("GetCurrentVersion", func() {
```

Comment:
Put a `#` sign in front of the function (`Context("#GetCurrentVersion")`) to align with the rest of the test suites.